bindings/python/iota_sdk/types/transaction_options.py

```diff
@@ -3,7 +3,8 @@
 
 from enum import Enum
 from typing import Optional, List, Union
-from dataclasses import dataclass
+from dataclasses import dataclass, field
+from iota_sdk.types.address import Address
```

Comment:
closing the gaps again I see :trollface:

Reply:
There was no gap?
Also if you ask me it's stupid to do if the formatter doesn't do it

Reply:
well it's also stupid to add two empty lines (and the formatter does THAT), but you're there was only a gap line in the display on GH, not in the code.

Reply:
Yes the two empty lines are really stupid